src/main/java/io/cryostat/rules/RuleProcessor.java

```diff
@@ -120,91 +118,84 @@ public synchronized void disable() {
     public synchronized void onEvent(Event<RuleEvent, Rule> event) {
         switch (event.getEventType()) {
             case ADDED:
+                platformClient.listDiscoverableServices().stream()
+                        .filter(serviceRef -> registry.applies(event.getPayload(), serviceRef))
+                        .forEach(serviceRef -> activate(event.getPayload(), serviceRef));
                 // FIXME the processor should also be able to apply new rules to targets that have
```

Comment:
What else would need to be changed in order to address the `fixme`?

Reply:
Ah - nothing. Whoops.